### PR TITLE
feat: Adopt Vert.x

### PR DESCRIPTION
**Pull request by thced**

### CODE REVIEW

File: radar/dev/vertx.yaml [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
+  - date: 2021-03-04
+    ring: ADOPT
 description: |
   Eclipse Vert.x is a polyglot event-driven application framework that runs on the
   Java Virtual Machine.
 
   Vert.x contains several different components designed to make it easier for
-  you to write compelling reactive applications in a range of different languages.
+  you to write reactive applications in a range of different languages.
   It is a compelling approach to writing microservices!
```
Comment (l0kal) [inline radar/dev/vertx.yaml:14]: Can you add a reason why it is compelling? I do not know the tool enough to get it from this context. :)

**Review by l0kal — CHANGES_REQUESTED** [review]
Left a comment on the description. It might be that I have just spent too much time in dotnet-land.

File: radar/dev/vertx.yaml [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
 
-  As of version 3.6.0, Vert.x exposes its API in Java, JavaScript, Groovy, Ruby,
-  Scala, Kotlin and Ceylon.
+  As of version 4.0.2, Vert.x exposes its API in Java, JavaScript, Groovy, Ruby,
+  Scala and Kotlin.
 rationale: |
   Vert.x is a very effective, highly performant framework where focus is on writing
   small isolated classes that communicates via a shared bus; EventBus.
```
Comment (sirenko7) [inline radar/dev/vertx.yaml:20]: Should be changed to "classes that communicate"

**Review by sirenko7 — CHANGES_REQUESTED** [review]

File: radar/dev/vertx.yaml (hunk 2 above)
Comment (thced) [reply radar/dev/vertx.yaml]: Fixed

**Review by sirenko7 — APPROVED** [review]

**Review by l0kal — APPROVED** [review]

**Pull request merged**